frontend/tests/unit/tests/work_packages/directives/inplace-editor-directive-test.js

```diff
@@ -54,7 +54,9 @@ describe('inplaceEditor Directive', function() {
                     'openproject.layout',
                     'openproject.config',
                     'openproject.services'));
-  beforeEach(inject(function($rootScope, $compile, _$timeout_, _WorkPackageService_) {
+  beforeEach(inject(function(
+    $rootScope, $compile, _$timeout_,
+    _WorkPackageService_, _OverviewTabInplaceEditorConfig_) {
```

Comment:
Identifier '_WorkPackageService_' is not in camel case.<br>Identifier '_OverviewTabInplaceEditorConfig_' is not in camel case.